src/Codeception/Lib/ModuleContainer.php

```diff
@@ -275,12 +280,37 @@ public function hasModule($moduleName)
     public function getModule($moduleName)
     {
         if (!$this->hasModule($moduleName)) {
-            throw new ModuleException(__CLASS__, "Module $moduleName couldn't be connected");
+            $this->throwMissingModuleExceptionWithSuggestion(__CLASS__, $moduleName);
         }
 
         return $this->modules[$moduleName];
     }
 
+    public function throwMissingModuleExceptionWithSuggestion($className, $moduleName)
+    {
+        $suggestedModuleNameInfo = $this->getModuleSuggestion($moduleName);
+        throw new ModuleException($className, "Module $moduleName couldn't be connected$suggestedModuleNameInfo");
```

Comment:
I really don't like that there is no space in `connected$suggestedModuleNameInfo`.
String concatenation would look better.